setup.py

```diff
@@ -36,7 +36,7 @@
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
-    install_requires=['PasteScript>=1.6'],
```

Comment:
Urx, there is no way to overwrite this kind of version pin. We tried to stay away from it in nearly every case because it causes more trouble than it solves.